package.json

```diff
@@ -20,7 +24,7 @@
     "webrtc-adapter-test": "^0.2.7"
   },
   "scripts": {
-    "test": "eslint rtcstats.js",
+    "test": "./node_modules/.bin/eslint rtcstats.js",
```

Comment:
Why?   ./node_modules/.bin/eslint is installed with npm install and this is a relative path that should be found, right?